target/linux/brcm63xx/image/bcm63xx.mk

```diff
@@ -56,8 +56,9 @@ DEVICE_VARS += REDBOOT_PREFIX
 ### Generic ###
 define Device/963281TAN-generic
   $(Device/bcm63xx)
+  DEVICE_VENDOR := Generic
+  DEVICE_MODEL := 963281TAN
   IMAGES := cfe-4M.bin cfe-8M.bin cfe-16M.bin
-  DEVICE_TITLE := Generic 963281TAN
```

Comment:
If you look at the block name, it looks to me as if "Generic" is used as an adjective here. So it would be wrong to use it as vendor, it is more like a variant. The question is what the vendor will be in this case.

Will have to think about that, maybe someone can illuminate us on a good solution for those devices.

Reply:
There is no good solution I think. These are e.g. images for devices using the "963281TAN" Board ID, with no idea which actual devices these are. There is a reference board using it ("BCM963281TAN"), and there are actual off-the-shelf devices using it, probably based on it. It would require quite a bit of research to find out which devices share the Board ID, and which of these also actually work.

The 4/8/16M images are images for the initial flash to prevent the bootloader from flashing it to the alternative image location. For sysupgrading, smaller images work fine.

Reply:
@KanjiMonster What's the manufacturer of the reference boards and would it make sense to put that name as DEVICE_VENDOR, so we keep the DEVICE_TITLE as new DEVICE_VENDOR?

Reply:
The reference board vendor would be "Broadcom". I don't think anyone else produces reference/evaluation boards based on Broadcom SoCs. And the name should still imply "generic image/target for devices using the 963281TAN board name", not claiming a certain vendor/device.

Reply:
@KanjiMonster 
In this case, I would just do the following:
DEVICE_VENDOR := Broadcom
DEVICE_MODEL := Generic 963281TAN

So, this will work perfectly with the new scheme, but we are not really changing anything (as DEVICE_TITLE will just be moved to DEVICE_MODEL).

Reply:
Won't this cause the model name to become "Broadcom Generic 963281TAN"? There should be no Broadcom in there.

Reply:
> Won't this cause the model name to become "Broadcom Generic 963281TAN"?

Yes. If you don't like that, I'd stick to mwarning's approach ...

Reply:
Yes, please keep the title as generic foo, not broadcom generic foo.